server/index.js

```diff
@@ -34,18 +34,21 @@ app.get('/', (req, res) => {
 
 // post route for incoming event packages (route name pending consensus with Event service)
 app.post('/view', (req, res) => {
-  var events = req.body.events;
+  var events = JSON.parse(req.body.body).events;
```

Comment:
I would consider making your app.use(expressWinston... all one line so you can group with the other app.use functions. Makes it easier on the eyes. 

Maybe consider naming your variable 'dbData' something a bit more descriptive. 

Looks great overall though!